code/game/objects/machinery/squad_supply/orbital_bombardement_beacon.dm

```diff
@@ -0,0 +1,104 @@
+
+/obj/item/orbital_bombardment_beacon
+	name = "orbital beacon"
+	desc = "A bulky device that fires a beam up to an orbiting vessel to send local coordinates."
+	icon_state = "motion4"
+	w_class = WEIGHT_CLASS_SMALL
+	var/activated = 0
+	var/activation_time = 80
+	var/datum/squad/squad = null
+	var/icon_activated = "motion1"
+	var/obj/machinery/camera/beacon_cam = null
```

Comment:
seeing those are similar vars on deploying, think it woulnd't be better to make a parent for both of them with activated, activation time, icon_activated, etc ? just for not repeating code.